Library/Homebrew/test/support/helper/spec/shared_context/integration_test.rb

```diff
@@ -78,7 +78,8 @@ def brew(*args)
       "HOMEBREW_BREW_FILE"        => HOMEBREW_PREFIX/"bin/brew",
       "HOMEBREW_INTEGRATION_TEST" => command_id_from_args(args),
       "HOMEBREW_TEST_TMPDIR"      => TEST_TMPDIR,
-      "HOMEBREW_DEVELOPER"        => ENV["HOMEBREW_DEVELOPER"],
+      "HOMEBREW_DEVELOPER"        => "1",
+      "HOMEBREW_RUBY_PATH"        => ENV["HOMEBREW_RUBY_PATH"],
```

Comment:
Why is this change needed? I'd be intrigued if this has any impact on https://github.com/Homebrew/brew/issues/6579.

Reply:
> I'd be intrigued if this has any impact on #6579.

I am not sure, but I don't think they are related.